ocaml/sdk-gen/go/gen_go_binding.ml

```diff
@@ -55,7 +55,15 @@ let main destdir =
         render_template "FileHeader.mustache" obj ~newline:true ()
       in
       let record_rendered = render_template "Record.mustache" obj () in
-      let rendered = header_rendered ^ record_rendered in
+      let methods_rendered = render_template "Methods.mustache" obj () in
+      let rendered =
+        let first_half = header_rendered ^ record_rendered in
```

Comment:
Sorry, just a question, we do not need a "\n" between `header_rendered` and `record_rendered`?

Reply:
No, after loop of `header_rendered `and `record_rendered`, it would add a "\n".